README.md

```diff
@@ -8,7 +8,42 @@ components.
 
 You will need at least the following installed:
 
- * LLVM 15.
- * wasm-ld 15 (this is part of LLVM, but your distribution might split it into
-   a separate package: your wasm-ld version must match your LLVM version).
- * typescript 5.2 or later.
+ * LLVM 15. Depending on your distribution you may need to install `llvm-devel`
+   or `llvm-extra` to get necessary extra parts such as `wasm-ld`.
+ * typescript (`tsc`) 5.2 or later. If your distribution doesn't package
+   typescript, you can install your own with `npm install
+   --prefix=/path/to/install typescript`.
+ * `yarn`
+
+
+## Build instructions
+
+First build and install the bootstrap compiler:
+
+```
+cd /path/to/repo/compiler && make clean && prefix=/path/to/install make install STAGE=0
```

Comment:
Why make clean?

Reply:
Probably that fault of my suggestion. For a fresh checkout, clean is not needed. But if you make changes to the compiler or runtime after building, the Makefile will not notice and `make -C compiler` will be a no-op until it is cleaned.

Reply:
*face-palm*